pkg/process/utils.go

```diff
@@ -201,3 +203,16 @@ func stateName(v interface{}) string {
 	}
 	panic(errors.Errorf("invalid state %v", v))
 }
+
+func parseRawSignal(rawSignal string) (unix.Signal, error) {
+	// allow 0 as valid signal
+	s, err := strconv.Atoi(rawSignal)
+	if err == nil {
+		return unix.Signal(s), nil
+	}
+	signal, err := signal.ParseSignal(rawSignal)
```

Comment:
So in the end, `ParseSignal` will return `syscall.Signal`, which is an alias for `int`. In this PR we add `rawSignal` everywhere along with the old `int` signal. Would it be possible to parse the string signal earlier (somwhere on containerd/client side) and just pass parsed `int` signal? This way we don't need to change APIs everywhere.

Reply:
That's the situation that this is trying to avoid; in the LCOW case, client != runtime platform (runc runs on Linux, but containerd on Windows).